test/cypress/e2e/search.spec.js

```diff
@@ -0,0 +1,97 @@
+import accounts from '../../constants/accounts';
+import networks from '../../constants/networks';
+
+const ss = {
+  searchInput: '#autosuggest-input',
+  delegateResults: '.delegates-result',
+  transactionResults: '.transactions-result',
+  recentSearches: '.addresses-result',
+  clearSearchBtn: '.autosuggest-btn-close',
+  explorerAccountLeftBlock: '.explorer-account-left-block',
+  accountAddress: '.copy-title',
+  accountBalance: '.balance span',
+  delegateName: '.delegate-name',
+  transactionId: '.transaction-id .copy-title',
+  logoutBtn: '.logout',
+  dialogButtons: '.ok-button',
+  emptyResultsMessage: '.empty-message',
+};
+
+const getSearchesObjFromLS = () => JSON.parse(localStorage.getItem('searches'));
+
+describe('Search', () => {
+  it('Search for Lisk ID', () => {
+    cy.autologin(accounts.genesis.passphrase, networks.devnet.node);
+    cy.visit('/');
+    cy.get(ss.searchInput).click().type(`${accounts.delegate.address}{enter}`);
```

Comment:
I think we also need an alternative to this test without `{enter}` but with clicking on the autosuggestions. But right now there is a bug that it shows "No results found" instead of suggestions for Lisk ID.